lib/core/lib/ops/userOps.js

```diff
@@ -513,6 +557,43 @@ function buildUserOps (db/*, config */) {
       })
     },
 
+    /**
+     * List the teams to which the user belongs to
+     * Does not go recursively to parent teams
+     *
+     * @param  {Object}   params { id, organizationId, limit, page }
+     * @param  {Function} cb
+     */
+    listUserTeams: function listUserTeams ({ id, organizationId, page = 1, limit }, cb) {
+      Joi.validate({ id, organizationId, page, limit }, validationRules.listUserTeams, function (err) {
+        if (err) return cb(Boom.badRequest(err))
+
+        const pageLimit = limit || _.get(config, 'authorization.defaultPageSize')
+        const offset = (page - 1) * pageLimit
```

Comment:
Maybe is better do do a `const offset = (Math.max(page, 1) - 1) * pageLimit` it can save you some trouble plus it's a small change

Reply:
this is limited by joi validation, no need to protect with max